### ISSUE TITLE
Issue during installation for M1 mac

### ISSUE THREAD

**Issue by itsyogesh** [jaketae/storyteller#6]
Hi folks,

I was trying to install the required packages but it seems there is an issue with `mecab-python3`. Here is the error log I saw:

```
 × Running setup.py install for mecab-python3 did not run successfully.
  │ exit code: 1
  ╰─> [14 lines of output]
      /Users/yogesh/Projects/Playground/storyteller/.env/lib/python3.10/site-packages/setuptools/installer.py:27: SetuptoolsDeprecationWarning: setuptools.installer is deprecated. Requirements should be satisfied by a PEP 517 installer.
        warnings.warn(
      running install
      /Users/yogesh/Projects/Playground/storyteller/.env/lib/python3.10/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
        warnings.warn(
      running build
      running build_py
      creating build
      creating build/lib.macosx-13.1-arm64-cpython-310
      creating build/lib.macosx-13.1-arm64-cpython-310/MeCab
      copying src/MeCab/__init__.py -> build/lib.macosx-13.1-arm64-cpython-310/MeCab
      copying src/MeCab/cli.py -> build/lib.macosx-13.1-arm64-cpython-310/MeCab
      running build_ext
      error: [Errno 2] No such file or directory: 'mecab-config'
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: legacy-install-failure

× Encountered error while trying to install package.
╰─> mecab-python3

note: This is an issue with the package mentioned above, not pip.
hint: See above for output from the failure.
```

I am using an M1 Mac so I am not sure if there any additional steps I missed.

**Comment by jaketae** [comment]
Hey @itsyogesh, thanks for opening this issue.

Seems like the issue is coming from `mecab-python3`. Can you try running this?

```
pip install mecab-python3
```

Based on some quick search, it appears that the `TTS` package has a dependency on `mecab`, which is causing issues for M1 users. The [issue](https://github.com/coqui-ai/TTS/issues/1533) doesn't list a clear solution, but I'm willing to investigate it further.

Also, I'm in the middle of making this project `pip`-installable, so hopefully installing will get a lot easier. Let me know how this goes!

**Comment by itsyogesh** [comment]
Hey @jaketae, thanks for the response, seems like the issue was getting the `mecab` either installed from source or brew before installing `mecab-python3`. Here is where I found out.

https://github.com/SamuraiT/mecab-python3/issues/84

I've gotten it installed and working now. Thanks for all the help. I have added a [PR](https://github.com/jaketae/storyteller/compare/master...itsyogesh:storyteller:patch-1) for this too.
